roundwared/recording_collection.py

```diff
@@ -124,9 +124,8 @@ def _get_recording(self):
         # If there are no playlist_proximity assets available, but there are
         # played assets available.
         if len(self.playlist_proximity) == 0 and self.has_played():
-            p = Project.objects.get(id=int(self.request['project_id']))
             # Check if continuous is enabled for the project.
-            if p.is_continuous():
+            if (self.project.repeat_mode == "continuous"):
```

Comment:
Since you got rid of the only call to the `is_continuous` function here, it's probably best to delete it altogether?

```
$ grep "is_continuous" * -Rn
roundware/rw/models.py:109:    def is_continuous(self):
```